{
  "info": {
    "author": "Mohd. Shafikur Rahman",
    "author_email": "shafikshaon@gmail.com",
    "bugtrack_url": null,
    "classifiers": [
      "Environment :: Web Environment",
      "Intended Audience :: Developers",
      "License :: OSI Approved :: MIT License",
      "Operating System :: OS Independent",
      "Programming Language :: Python :: 2.7",
      "Programming Language :: Python :: 3"
    ],
    "description": "# django-debug-helper\nDjango debug error related exception search on Google/ Stack Overflow.\n\n### Installation\n```\n$ pip install django-debug-helper\n```\n\n### Setting it up\nAdd the custom middleware to your **MIDDLEWARE** in the ```settings.py```.\n```\nif DEBUG:\n\tMIDDLEWARE += ('django-debug-helper.debug.ErrorSearchMiddleware', )\n```\nNext time you get an error in your applications, you will see an external link to a custom **google/ stackoverflow** search as you defined.\n\n\nBy default it set to **google** search, but if you want to use **stackoverflow** search instead of **google** then add this into your ```settings.py```\n```\nDJANGO_TRACE_SEARCH_SITE = \"stackoverflow\"\n```\n\n\n\n",
    "description_content_type": "text/markdown",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/shafikshaon/django-debug-helper",
    "keywords": "",
    "license": "MIT",
    "maintainer": "",
    "maintainer_email": "",
    "name": "django-debug-helper",
    "package_url": "https://pypi.org/project/django-debug-helper/",
    "platform": "",
    "project_url": "https://pypi.org/project/django-debug-helper/",
    "project_urls": {
      "Homepage": "https://github.com/shafikshaon/django-debug-helper"
    },
    "release_url": "https://pypi.org/project/django-debug-helper/1.1.4/",
    "requires_dist": null,
    "requires_python": "",
    "summary": "Django debug error related exception search on Google/ Stack Overflow.",
    "version": "1.1.4",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 4471849,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "ea37d77e590497982d103a7137b6b87e76fcd230f43d8cd84628e5a9f1fdcc40",
        "md5": "9a475d69d601fbb8e5a2c09cc2f1138d",
        "sha256": "3395146b7f6f90ab98c7dd0d21720e03c3579ee40dab43c0c371909f649f7590"
      },
      "downloads": -1,
      "filename": "django_debug_helper-1.1.4-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "9a475d69d601fbb8e5a2c09cc2f1138d",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": null,
      "size": 4984,
      "upload_time": "2018-11-09T20:01:42",
      "upload_time_iso_8601": "2018-11-09T20:01:42.240030Z",
      "url": "https://files.pythonhosted.org/packages/ea/37/d77e590497982d103a7137b6b87e76fcd230f43d8cd84628e5a9f1fdcc40/django_debug_helper-1.1.4-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "3fb28dd97fc6d739b1a5eb73e349bedb412dcd92117714571c1ef956a3bc9549",
        "md5": "96fa199ee1a159fc03f8ee081a73f919",
        "sha256": "c8689888c44c451112446d569766e98b4d3b4b5892e47749d3f80cf856f2ccf7"
      },
      "downloads": -1,
      "filename": "django-debug-helper-1.1.4.tar.gz",
      "has_sig": false,
      "md5_digest": "96fa199ee1a159fc03f8ee081a73f919",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": null,
      "size": 2485,
      "upload_time": "2018-11-09T20:01:45",
      "upload_time_iso_8601": "2018-11-09T20:01:45.409825Z",
      "url": "https://files.pythonhosted.org/packages/3f/b2/8dd97fc6d739b1a5eb73e349bedb412dcd92117714571c1ef956a3bc9549/django-debug-helper-1.1.4.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}